lib/archive_importer.rb

```diff
@@ -36,8 +36,9 @@ def create_user(attr)
         profile_attributes: profile_attributes
       }
     )
-    self.user = User.build(data)
-    user.save!
+    @user = User.find_or_build(data)
+    @user.getting_started = false
```

Comment:
This is already set in the hash above, but it probably doesn't work because the `User.setup` method doesn't handle it. Instead of setting this separately here, I would just add it to the `setup` method, so the value from the hash works. This is basically the only "setting" I feel confident to always overwrite when migrating, because when a user imports an existing archive we can be sure it isn't a new user and doesn't need "getting started" anymore.